src/main/resources/swagger/api-docs.yaml

```diff
@@ -126,13 +126,12 @@ paths:
             - openid
             - email
             - profile
-  '/api/notebooks/{googleProject}/{clusterName}':
-    get:
-      summary: Access Jupyter notebooks on a dataproc cluster
-      description: Serves as a proxy for the Jupyer notebook server running the given dataproc cluster managed by Leo.
-      operationId: proxyCluster
+    delete:
```

Comment:
I think there may have been a swagger merge issue (probably my fault): the `delete` method was under `/api/notebooks/..` which is not correct. This fixes it.